tools/setup_workspace.py

```
@@ -1229,17 +1255,17 @@ def _search_runs(runs_url: str) -> tuple[str | None, str | None]:
                 print(Console.warn(t("wrn_ci_artifact_rate_limited", code=e.code)))
             else:
                 print(Console.warn(t("wrn_ci_artifact_list_runs_failed", error=e)))
-            return None, None
+            return None, None, None
         except urllib.error.URLError as e:
             print(Console.warn(t("wrn_ci_artifact_network_error", error=e.reason)))
-            return None, None
+            return None, None, None
         except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)